CHANGELOG.asciidoc

```diff
@@ -7,6 +7,52 @@
 === Beats version 8.4.1
 https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 
+==== Known Issue 
+
+// tag::credentials-error[]
+
+*Filebeat*
+
+Filebeat agents configured to read from S3 or SQS may return the following
```

Comment:
leaving here similar comment as ones at https://github.com/elastic/observability-docs/pull/2168
I would be "less" specific: meaning is a problem related to any AWS metricset/fileset and not specific to S3 and SQS.